types/is-data-descriptor/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for is-data-descriptor 1.0
+// Project: https://github.com/jonschlinkert/is-data-descriptor
+// Definitions by: Richie Bendall <https://github.com/Richienb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function isDataDesc(obj: unknown): obj is PropertyDescriptor;
+declare function isDataDesc(obj: unknown, key: unknown): boolean;
```

Comment:
This should be something like:
```ts
declare function isDataDesc<T extends object>(obj: T, key: keyof T): boolean;
```

Could you please also add a test for this overload?

Reply:
Not sure if the linter will like `object`

Reply:
Try it out.